src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -19,7 +19,7 @@ public int getSize() {
 	@Override
 	public void push(int in) {
 		if (!isFull())
-			numbers[++size] = in;
+			numbers[size++] = in;
```

Comment:
Changing IS_EMPTY value requires such fixes? It does not make sense. Incorrect fix.